generate_parameter_library/cmake/generate_parameter_library.cmake

```diff
@@ -48,37 +48,46 @@ function(generate_parameter_library LIB_NAME YAML_FILE)
     cmake_path(APPEND VALIDATE_HEADER ${VALIDATE_HEADER_FILENAME})
 
     # Copy the header file into the include directory
-    add_custom_command(
-      OUTPUT ${VALIDATE_HEADER}
-      COMMAND ${CMAKE_COMMAND} -E copy ${IN_VALIDATE_HEADER} ${VALIDATE_HEADER}
-      DEPENDS ${IN_VALIDATE_HEADER}
-      COMMENT
-      "Running `${CMAKE_COMMAND} -E copy ${IN_VALIDATE_HEADER} ${VALIDATE_HEADER}`"
-      VERBATIM
-    )
+    file(COPY ${IN_VALIDATE_HEADER} DESTINATION ${LIB_INCLUDE_DIR})
+    # necessary so that #include <param_file.hpp> can be used in the local package (deprecated)
+    file(COPY ${IN_VALIDATE_HEADER} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/include)
```

Comment:
thank you for keeping this around for compatibility!

Reply:
You're welcome !